app/javascript/onboarding/components/FollowTags.jsx

```diff
@@ -130,53 +130,53 @@ export class FollowTags extends Component {
-                <div
-                  className={`onboarding-tags__item ${
-                    selectedTags.includes(tag) &&
-                    'onboarding-tags__item--selected'
-                  }`}
-                  style={{
-                    boxShadow: selectedTags.includes(tag)
-                      ? `inset 0 0 0 100px ${tag.bg_color_hex}`
-                      : `inset 0 0 0 2px ${tag.bg_color_hex}`,
-                    color: selectedTags.includes(tag) ? tag.text_color_hex : '',
-                  }}
-                >
-                  <div className="onboarding-tags__item__inner">
-                    #{tag.name}
-                    <button
-                      type="button"
-                      onClick={() => this.handleClick(tag)}
-                      className={`onboarding-tags__button ${
-                        selectedTags.includes(tag) &&
-                        'onboarding-tags__button--selected'
-                      }`}
-                      style={{
-                        backgroundColor: selectedTags.includes(tag)
-                          ? tag.text_color_hex
-                          : tag.bg_color_hex,
-                        color: selectedTags.includes(tag)
-                          ? tag.bg_color_hex
-                          : tag.text_color_hex,
-                      }}
-                    >
-                      {selectedTags.includes(tag) ? (
-                        <span>
-                          <span className="onboarding-tags__button-default">
-                            ✓ Following
-                          </span>
-                          <span className="onboarding-tags__button-alt">
-                            Unfollow
-                          </span>
-                        </span>
-                      ) : (
-                        'Follow'
-                      )}
-                    </button>
+              {allTags.map((tag) => {
+                const selected = selectedTags.includes(tag);
+                return (
+                  <div
+                    className={`onboarding-tags__item ${
+                      selected && 'onboarding-tags__item--selected'
+                    }`}
+                    style={{
+                      boxShadow: selected
+                        ? `inset 0 0 0 100px ${tag.bg_color_hex}`
+                        : `inset 0 0 0 2px ${tag.bg_color_hex}`,
+                      color: selected ? tag.text_color_hex : '',
+                    }}
+                  >
+                    <div className="onboarding-tags__item__inner">
+                      #{tag.name}
+                      <button
+                        type="button"
+                        onClick={() => this.handleClick(tag)}
+                        className={`onboarding-tags__button crayons-btn--icon-left ${
+                          selected && 'onboarding-tags__button--selected'
+                        }`}
```

Comment:
See below - I think we only want this class when the button is selected

```suggestion
                        className={`onboarding-tags__button  ${
                          selected && 'onboarding-tags__button--selected crayons-btn--icon-left'
                        }`}
```

Reply:
This is how it looks after the changes

![Tags](https://user-images.githubusercontent.com/8092120/116885671-c7382b00-ac45-11eb-8ea8-ace5a1ccc249.gif)